Makefile

```diff
@@ -46,6 +46,6 @@ push.image: build.image
 push.image.rhel8: build.image.rhel8
 	$(CONTAINER_ENGINE) push ${IMG}:${IMAGE_TAG}
 
-deploy: push.image.rhel8
-	oc new-project node-observability-operator
+deploy: #push.image.rhel8
```

Comment:
Let's not keep code that's commented out

Reply:
Thanks! 
I commented it to test quicker :( 
Just put it back